tests/models/vivit/test_modeling_vivit.py

```diff
@@ -363,9 +363,7 @@ def test_inference_interpolate_pos_encoding(self):
 
         image_processor = VivitImageProcessor.from_pretrained("google/vivit-b-16x2")
         video = prepare_video()
-        inputs = image_processor(
-            video, size={"shortest_edge": 480}, crop_size={"height": 480, "width": 480}, return_tensors="pt"
```

Comment:
crop_size option should still be included in the test, as this will force the interpolation

Reply:
sure, let me push the commit in a second

Reply:
done :+1: